doc/development_process/dev_env_and_tools.rst

```diff
@@ -268,12 +268,22 @@ To be discussed in a meeting
 ============================
 
 =============  ===============================================================
-Labels         ``API``, ``dev-review``, ``TSC``
-Applicable to  PRs  and issues
+Labels         ``meeting: {API|dev-review|TSC}``
+Applicable to  PRs and issues
 Description    The issue is to be discussed in the following
                `API/dev-review/TSC meeting`_ if time permits
 =============  ===============================================================
 
+Normally, PRs and issues are discussed and resolved directly in GitHub.
```

Comment:
A bit of cleanup and removing the male pronoun :)
```
Normally, PRs and issues are discussed and resolved directly in GitHub.
In some cases, further discussion may be needed in one of the
periodic `project meetings`_. One of these labels should be added
to the PR or issue by the author or maintainer to bring it to the
attention of that forum. The author is expected to attend that
meeting to participate in the discussion.
```

How does the author know if that issue/PR will be on the agenda?

Reply:
Thanks @dbkinder , fixes (I took the liberty of adding an extra article before maintainer, as there is not 1 maintainer per PR)

> How does the author know if that issue/PR will be on the agenda?

Today, the author cannot really know (but can try to force it by speaking up during the meeting)